src/afrolabs/components/kafka.clj

```diff
@@ -2096,7 +2125,7 @@
     "Waits until a ktable has consumed messages from its source topics, at least up to the topic-partition-offset data parameter.
 Supports a timeout operation. `timeout-duration` must be a java.time.Duration.")
   (ktable-wait-until-fully-current
-    [_this timeout-period timeout-value]
+    [_this timeout-duration timeout-value]
```

Comment:
in `java.time` there is a distinction between `Period` and `Duration`. (Periods are measured in calender units, days, months, weeks) whereas Duration is measured in milliseconds.